### ISSUE TITLE
How to upload remote s3 videos to cvat for annotation in YOLO format using cvat sdk or cvat api?

### ISSUE THREAD

**Issue by spannenberger** [opencv/cvat#6238]
### My actions before raising this issue
- [ ] Read/searched [the docs](https://github.com/cvat-ai/cvat/tree/master#documentation)
- [ ] Searched [past issues](/issues)

### Steps to Reproduce (for bugs)

1.
1.
1.
1.

### Context

I have about 500 video fragments every week and now the markup process looks like this: I unload from the bucket to the local machine, manually upload the video to cvat with UI, and this is very long. I want to write python script which can load videos from s3 bucket and than upload them into cvat in new project with YOLO annotations
### Your Environment

- Git hash commit (`git log -1`):
- Docker version `docker version` (e.g. Docker 17.0.05):
- Are you using Docker Swarm or Kubernetes?
- Operating System and version (e.g. Linux, Windows, MacOS):
- Code example or link to GitHub repo or gist to reproduce problem:
- Other diagnostic information / logs:
    <details>
    <summary>Logs from `cvat` container</summary>
    </details>

**Comment by zhiltsov-max** [comment]
Hi! Please take a look at these examples:
- [Task creation with high-level SDK](https://github.com/opencv/cvat/blob/develop/tests/python/sdk/test_tasks.py#L130-L145)
- [Task creation with s3 data and low-level SDK](https://github.com/opencv/cvat/blob/develop/tests/python/rest_api/test_tasks.py#L819-L872)
- [Importing annotations with high-level SDK](https://github.com/opencv/cvat/blob/develop/tests/python/sdk/test_tasks.py#L351-L360)
- [Creating a task with CLI](https://opencv.github.io/cvat/docs/api_sdk/cli/#create)

Basically, the following code snippet should be close to what you're looking for:

```python
from cvat_sdk import make_client, models
from cvat_sdk.core.proxies.tasks import ResourceType

with make_client("localhost") as client:
    client.login(("username", "password"))

    task = client.tasks.create_from_data(models.TaskWriteRequest(
        name="my task",
        labels=[{"name": "cat"}, {"name": "dog"}],
    ), resources=[
        "server filename 1.mp4"
    ], resource_type=ResourceType.SHARE)
```
